src/components/Footer/Footer.js

```diff
@@ -3,28 +3,77 @@ import Gryffindor from "../img/Gryffindor.jpg";
 import Ravenclaw from "../img/Ravenclaw.jpg";
 import Slytherin from "../img/Slytherin.jpg";
 import Hufflepuff from "../img/Hufflepuff.jpg";
+import { useState } from "react";
 
 function Footer() {
+  const [button, setButtonColor] = useState("");
+
   return (
     <div className="Footer__styling">
-      <img class="Footer__button" src={Gryffindor} />
-      <img class="Footer__button" src={Ravenclaw} />
-      <img class="Footer__button" src={Hufflepuff} />
-      <img class="Footer__button" src={Slytherin} />
+      <img
+        className={`Footer__button ${
+          button === "Gryffindor" ? "footer__button--activered" : ""
+        }`}
+        onClick={() => setButtonColor("Gryffindor")}
+        class="Footer__button"
+        src={Gryffindor}
+      />
```

Comment:
This should be a `button` element! Or a `button` around the `img` element. And you are missing the `alt` attribute.